.github/workflows/e2e-matrix.yaml

```
@@ -63,31 +63,44 @@ jobs:
         suite:
           - name: IPv6
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: AMI
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Scheduling
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Storage
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Integration
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: NodeClaim
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Consolidation
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Interruption
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Drift
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Expiration
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Chaos
             region: ${{ inputs.region }}
+            source: aws_provider
           - name: Termination
             region: ${{ inputs.region }}
+            source: aws_provider
```

```suggestion
            source: aws
```
nit: I'd just call it `aws` and `upstream`